pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/custom/GeoSpatialTest.java

```diff
@@ -309,7 +309,9 @@ public void testStPointFunctionWithV2(boolean useMultiStageQueryEngine)
                   + "b.st_point, "
                   + "b.st_point_1, "
                   + "ST_DISTANCE(ST_Point(a.st_x, a.st_y, -1), b.st_point), "
```

Comment:
Not introduced in this PR, but this should be 0 instead of -1. -1 is not a valid representation of boolean

Reply:
I was checking the Boolean utils, any value other than 1 is treated as false, hence the test.

Reply:
That is due to the current implementation. In SQL standard, any number other than 0 is treated as `true`. The fix for this behavior: #11763